package.json

```diff
@@ -26,44 +26,53 @@
   "author": "4DN DCIC",
   "license": "MIT",
   "devDependencies": {
+    "@babel/core": "^7.17.5",
+    "@babel/eslint-parser": "^7.5.0",
+    "@babel/plugin-proposal-class-properties": "^7.16.7",
+    "@babel/plugin-proposal-object-rest-spread": "^7.17.3",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-react": "^7.16.7",
+    "@babel/register": "^7.17.0",
+    "@babel/runtime": "^7.17.8",
+    "@babel/plugin-transform-runtime":"^7.17.0",
+    "@babel/plugin-proposal-export-default-from":"^7.16.7",
+    "@babel/plugin-transform-template-literals":"^7.16.7",
+    "@babel/plugin-proposal-decorators":"^7.17.8",
+    "@babel/plugin-proposal-pipeline-operator":"^7.17.6",
+    "@babel/plugin-transform-async-to-generator":"^7.16.8",
+    "babel-jest": "^27.5.1",
+    "babel-loader": "^8.2.3",
+    "babel-plugin-minify-dead-code-elimination": "^0.5.1",
     "@babel/cli": "^7.6.0",
-    "@babel/core": "^7.1.2",
-    "@babel/plugin-proposal-class-properties": "^7.3.3",
-    "@babel/plugin-proposal-object-rest-spread": "^7.3.2",
-    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-    "@babel/preset-env": "^7.1.0",
-    "@babel/preset-react": "^7.0.0",
-    "@babel/register": "^7.0.0",
-    "babel-eslint": "^10.0.1",
-    "babel-loader": "^8.0.4",
-    "babel-plugin-minify-dead-code-elimination": "^0.5.0",
-    "d3": "^5.9.0",
-    "eslint": "^5.16.0",
-    "eslint-plugin-react": "^7.16.0",
+    "d3": "^6.7.0",
+    "eslint": "^8.10.0",
+    "eslint-plugin-cypress": "^2.12.1",
+    "eslint-plugin-react": "^7.29.2",
     "fancy-log": "^1.3.3",
     "gulp": "^4.0.0",
     "http-server": "^0.12.1",
     "jasmine": "^3.5.0",
     "sass": "^1.39.0",
     "plugin-error": "^1.0.1",
     "prop-types": "^15.7.2",
-    "react": ">=16.14.0",
-    "react-dom": ">=16.14.0",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
     "react-transition-group": "^4.4.1",
     "source-map-support": "^0.5.19",
     "string-replace-loader": "^2.3.0",
     "terser-webpack-plugin": "^4.2.3",
-    "webpack": "4.44.2"
+    "webpack": "^4.46.0"
   },
   "peerDependencies": {
-    "d3": ">=5.9.0",
+    "d3": "^6.7.0",
```

Comment:
Should revert this to `"d3": ">=5.9.0"` as would allow older versions to still work ([CGAP is on v5 still](https://github.com/dbmi-bgm/cgap-portal/blob/master/package.json#L89)) while remaining compatible with v6+.